Makefile

```diff
@@ -1,29 +1,24 @@
 .PHONY: install-web
 install-web:
-	cd ./packages/web/ && npm install
+	cd ./services/web/ && npm install
 
 .PHONY: run-web
 run-web:
-	cd ./packages/web/ && npm run start
+	cd ./services/web/ && npm run dev
 
-.PHONY: run-dev
-run-dev:
-	cd ./packages/web/ && npm run dev
+# TO-DO: Write the script for spinning up the dev server
+# .PHONY: run-dev
+# run-dev:
+# 	cd ./services/web/ && npm run start
 
 .PHONY: run-prod
 run-prod:
-	cd ./packages/web/ && npm run prod
-
-.PHONY: run-api
-run-api:
-	cd ./packages/api/ && go run main.go
+	cd ./services/web/ && npm run prod
```

Comment:
Do these commands still exist after @Aleks1908 reinitialized the project?
I recommend discarding all of those for now and adding them only as needed.